rotkehlchen/inquirer.py

```diff
@@ -139,6 +142,10 @@ def get_underlying_asset_price(token: EthereumToken) -> Optional[Price]:
     price = None
     if token.protocol == 'UNI-V2':
         price = Inquirer().find_uniswap_v2_lp_price(token)
+    elif token.protocol == 'curve_lp':
+        price = Inquirer().find_curve_lp_price(token)
+    elif token.protocol == 'yearn_vault':
```

Comment:
If these are to be hard-coded protocols they should be moved into a constant somewhere.